03_launch_mgmt_cluster.sh

```diff
@@ -341,6 +341,13 @@ function patch_clusterctl(){
   mkdir -p "${HOME}"/.cluster-api
   touch "${HOME}"/.cluster-api/clusterctl.yaml
 
+  ## Remove these lines
+  ## This is hard-coded until we fix the issue with v1.9.1
```

Comment:
This comment looks strange. If there is some issue with v1.9.1 why are we using it?

Reply:
Thats a wrong comment. Good catch, fixing it.

Reply:
+1, it could be rephrased a bit

Reply:
fixed